### ISSUE TITLE
node-red-node-piface  pre-reqs in readme wrong

### ISSUE THREAD

**Issue by colinl** [node-red/node-red-nodes#735]
The readme for node-red-node-piface has the following pre-reqs:
```
Device Tree must be turned off. To do this run

sudo raspi-config

then select the Advanced Options, then Device Tree, and finally select No and OK. You will then need to reboot.

It also requires the WiringPi gpio command to be installed in order to work. See the WiringPi site for details on how to do this. The short version is...

sudo apt-get install git-core
git clone git://git.drogon.net/wiringPi
cd wiringPi
git pull origin
./build
```
Unfortunately Device Tree appears to be no longer available in raspi-config and the git repository mentioned is not available.
I don't know what the correct procedure is, or I would submit a PR, I found this trying to help someone on the forum.

**Comment by dceejay** [comment]
Yes - I think it would need a total re-write to use the python RPi.GPIO (like the gpio nodes) - or probably the official one would be best - http://www.piface.org.uk/guides/Install_PiFace_Software/First_steps_with_PiFace_Digital_Python/

Sadly I no longer have a PiFace board so am not in a position to be able to take on this task.
